app/routes/events/components/EventDetail/index.js

```diff
@@ -111,7 +109,16 @@ type Props = {
   currentUserFollowing: ?FollowerItem,
 };
 
-export default class EventDetail extends Component<Props> {
+type State = { mapIsOpen: boolean };
+
+export default class EventDetail extends Component<Props, State> {
```

Comment:
This is also a fitting PR to turn this into a functional component 🤗

Reply:
I had quite a lot of troubling learning the class component haha, I'm all for functional